dist_tables/dml.rst

```diff
@@ -8,23 +8,25 @@ The following code snippets use the Github events example, see :ref:`ddl`.
 Inserting Data
 --------------
 
-Single row inserts
-$$$$$$$$$$$$$$$$$$
+Inserting Row Literals
```

Comment:
It's applicable to more than a single row, so just called it a row literal because you're inserting literal values directly.